### ISSUE TITLE
Frontend message: Something went wrong while connecting to the server. Please try again later

### ISSUE THREAD

**Issue by mallibus** [pieroit/cheshire-cat#53]
Cat running in a Virtualbox VM  ubuntu-20.04.6-live-server-amd64.iso, hosted by a Windows 10 Enterprise PC
Web interface starts on <VM IP address>:3000
Message "Getting Ready" and then red banner telling "Something went wrong while connecting to the server. Please try again later"

The only warning message on the log window is

```
web         | /usr/local/lib/python3.9/site-packages/langchain/llms/openai.py:608: UserWarning: You are trying to use a chat model. This way of initializing it is no longer supported. Instead, please use: `from langchain.chat_models import ChatOpenAI`
web         |   warnings.warn(
```

I'm not sure if it has anything to do with the problem.
maybe it is something super simple, but the message is not much helping. For example I'm not sure which server is not reachable. A more detailed message, at least on the log, with server name/address and port would help in troubleshooting.

**Comment by mallibus** [comment]
This just exposes how naive I am. To fix the specific issue I just had to configure the VM Networking in "NAT" mode and to add the rules to forward port 3000 and port 1865 from the localhost IP (127.0.0.1) to the VM IP (in my case the default 10.0.2.15)

**Comment by pieroit** [comment]
Sorry never used VMs!
Thank you for trying that out!

**Comment by mallibus** [comment]
no problem! I was fooled by the fact that initially I configured the VM networking as "bridged" and I was able to connect to the frontend with "<VM IP>:3000" so I supposed that as I was able to talk to the cat, the cat was able to connect outside. But something was wrong with that approach, and after some struggle I was able to fix switching to NAT and enabling the port forwarding for the ports used by the cat.
